<ui version="4.0" >
 <class>TransDialog1D</class>
 <widget class="QDialog" name="TransDialog1D" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>1D Transfer Function Editor</string>
  </property>
  <property name="locale" >
   <locale country="UnitedStates" language="English" />
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_Components" >
     <property name="minimumSize" >
      <size>
       <width>131</width>
       <height>141</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>131</width>
       <height>141</height>
      </size>
     </property>
     <property name="locale" >
      <locale country="UnitedStates" language="English" />
     </property>
     <property name="title" >
      <string>Color Components</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="checkBox_Red" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Red</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Green" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>50</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Green</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Blue" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Alpha" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>110</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Alpha</string>
      </property>
     </widget>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" >
    <widget class="QGraphicsView" name="graphicsView_Histogram" >
     <property name="minimumSize" >
      <size>
       <width>309</width>
       <height>307</height>
      </size>
     </property>
     <property name="locale" >
      <locale country="UnitedStates" language="English" />
     </property>
     <property name="renderHints" >
      <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="2" >
    <widget class="QSlider" name="verticalSlider_ScaleHistogram" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_Groups" >
     <property name="minimumSize" >
      <size>
       <width>131</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>131</width>
       <height>111</height>
      </size>
     </property>
     <property name="title" >
      <string>Groups</string>
     </property>
     <widget class="QRadioButton" name="radioButton_User" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>User</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Luminance" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>50</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Luminance</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Intensity" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Intensity</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>128</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="locale" >
      <locale country="UnitedStates" language="English" />
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_Red</tabstop>
  <tabstop>checkBox_Green</tabstop>
  <tabstop>checkBox_Blue</tabstop>
  <tabstop>checkBox_Alpha</tabstop>
  <tabstop>radioButton_User</tabstop>
  <tabstop>radioButton_Luminance</tabstop>
  <tabstop>radioButton_Intensity</tabstop>
  <tabstop>graphicsView_Histogram</tabstop>
  <tabstop>verticalSlider_ScaleHistogram</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Red</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetUserMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Green</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetUserMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Blue</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetUserMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Alpha</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetUserMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Intensity</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetIntensityMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>60</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Luminance</sender>
   <signal>clicked()</signal>
   <receiver>TransDialog1D</receiver>
   <slot>SetLuminanceMode()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>60</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetUserMode()</slot>
  <slot>SetLuminanceMode()</slot>
  <slot>SetIntensityMode()</slot>
 </slots>
</ui>
